dtos/tags.go

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package dtos
+
+import "encoding/xml"
+
+// Tags is the alias of map[string]any type
```

Comment:
This comment states the obvious. Please only add comments that explain why when code it to complicated to understand be reading it.